plugins/dialog/plugin.js

```diff
@@ -2107,6 +2128,37 @@ CKEDITOR.DIALOG_STATE_BUSY = 2;
 		ev.data.preventDefault( 1 );
 	}
 
+	var document = CKEDITOR.document,
+		body,
+		scrollBarWidth,
+		paddingRight;
+
+	function handleBodyStyles( dialogOpen ) {
```

Comment:
Instead of passing `dialogOpen` flag, I would split this function into two separate helpers. This function is used only in two places with strictly passed boolean value, so there is no need for such complication. Also, smaller but verbose functions don't need additional comments. What about:

`addDocumentBodyStyles`
`removeDocumentBodyStyles`